install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,34 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes have labels that often indicate their region. A convention is to set the
+`*region=infra*` label for the router and registry selector. For example:
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
+
+The default label is `*region=infra*`. In all other aspects, this is a normal
+node. Not all nodes are running mission-critical containers. While a node
+labeled `*dev*` might be rebooted or deleted at any time, the `*infra*` nodes
```

Comment:
A label comprises key and value, so "labelled *dev*" is incomplete (likewise, naked *infra*).  For the latter, i suppose the key would be *region* (thus, "the `*region=infra*` nodes").  Maybe the same key is used for *dev*, but i'm guessing not.